### PR TITLE
BZ1468718 Add Scaling the Cassandra Components Instructions to OCP 3.2

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1468718

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation Please peer review 🎆

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -668,23 +668,80 @@ link:https://github.com/kubernetes/heapster/[Heapster] project.
...
+
+----
+# oc scale -n openshift-infra --replicas=2 rc hawkular-metrics
+----
+endif::openshift-enterprise[]
+
+[NOTE]
+====
+If you add a new node to a Cassandra cluster, the data stored in the cluster
+rebalances across the cluster. The same thing happens If you remove a node from
```
Comment (tmorriso-rh) [inline install_config/cluster_metrics.adoc]: The same thing happens if you remove a node from the cluster.

**Comment by bmcelvee** [comment]
Thanks @tmorriso-rh! 😸

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
|Added the xref:../install_config/cluster_metrics.adoc#cluster-metrics-scaling-openshift-metrics-pods[Scaling {product-title} Metrics Pods] section.
%

**Pull request merged**